packages/internal-lib-build/configs/babel.config.js

```diff
@@ -29,7 +29,8 @@ const config = {
                 idInterpolationPattern: '[sha512:contenthash:base64:6]',
                 ast: true
             }
-        ]
+        ],
+        require('babel-plugin-transform-html-import-to-string')
```

Comment:
Given that we're trying to limit the number. of dependencies we have, it doesn't make sense to me to add a new one to serve a piece of static HTML. We can already do that with Express.